C/Cpp_include/access_lock.hh

```diff
@@ -167,56 +174,50 @@ namespace litecore {
         template <class RESULT, class LAMBDA>
         RESULT useLocked(LAMBDA callback) {
             LOCK_GUARD lock(_mutex);
-            if (_sentry) _sentry(_contents);
+            if ( _sentry ) _sentry(_contents);
             return callback(_contents);
         }
 
         // const versions:
 
-        auto useLocked() const  {
-            return access<const T&>(getMutex(), _contents, const_cast<SENTRY&>(_sentry));
-        }
+        auto useLocked() const { return access<const T &>(getMutex(), _contents, const_cast<SENTRY &>(_sentry)); }
```

Comment:
How long are lines allowed to be in this format?

Reply:
120. Most standards have it at 80, but this feels quite strict (especially with higher resolution displays becoming increasingly common)